recipes/nsis/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.46" %}
+
+package:
+  name: nsis
+  version: {{ version }}
+
+source:
+  fn: nsis-{{ version }}.tar.bz2
+  url: http://sourceforge.net/projects/nsis/files/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}-src.tar.bz2
+  sha256: f5f9e5e22505e44b25aea14fe17871c1ed324c1f3cc7a753ef591f76c9e8a1ae
+
+build:
+  number: 0
+  skip: true  # [unix]
+  script:
+    # Hack added to avoid issues with `scons`.
+    #
+    # Appears `scons` expects to find another
+    # path because of the trailing `;` in
+    # `LIB`. This changes it to not have a
+    # trailing `;` so as to avoid these issues.
+    - set "LIB=%LIBRARY_LIB%"
```

Comment:
Sorry, this one is a bear, and I don't really know where to start.  LIB is definitely set, and scons seems to be misbehaving.  I think you need to start with outputting all of the environment variables prior to starting the scons build process.  A line with:

```
set
```

should do it.